internal/tasks/process-file-updates-task/process-file-updates-task.yaml

```diff
@@ -243,7 +243,61 @@ spec:
         fi
 
         echo -e "\n*** START LOCAL CHANGES ***\n"
-        git diff
+        git diff | tee "${TEMP}"/tempMRFile.diff
+
+        if [[ -s "${TEMP}"/tempMRFile.diff ]]; then
+            # replacements exist
+            awk '/^@@/ {match($0, /@@ ([^@]+) @@/, arr); print arr[1]}' "${TEMP}"/tempMRFile.diff |\
```

Comment:
What does this do? Can you add a comment explaining what it will print?

Reply:
It's to deal with a line like "@@ -4,7 +4,7 @@ $schema: /app-interface/app-interface-settings-1.yml".  Usually it should be "@@ -4,7 +4,7 @@" in a line and " $schema: /app-interface/app-interface-settings-1.yml" in a seperated line.

Reply:
I still don't get it :joy: but can you make that comment in the code? This will be lost once this is merged